script/security-check.js

```diff
@@ -11,6 +11,7 @@ const packageJSON = require('../package.json');
 const exceptionSet = new Set([
   'https://npmjs.com/advisories/577',
   'https://npmjs.com/advisories/157',
+  'https://www.npmjs.com/advisories/788',
```

Comment:
We had to add an exception here anyway, because Metalsmith uses it too. I wonder if we can issue a PR to Metalsmith to upgrade it.

```
Security advisory:
  Title: Denial of Service
  Module name: js-yaml
  Dependency: metalsmith
  Path: metalsmith>gray-matter>js-yaml
  Severity: moderate
  Details: https://npmjs.com/advisories/788
```